test/helpers/sign.js

```diff
@@ -11,12 +17,57 @@ export const hashMessage = (message) => {
   return utils.bufferToHex(utils.sha3(Buffer.concat([prefix, messageHex])));
 };
 
-// signs message using web3 (auto-applies prefix)
-export const signMessage = (signer, message = '', options = {}) => {
-  return web3.eth.sign(signer, web3.sha3(message, options));
```

Comment:
Why this change? I guess it's related to `web3.sha3` vs `soliditySha3`... What's the difference between those two?

Reply:
soliditySha3 abi encodes and then tightly-packs arguments and sha3 just operates on a buffer or whatever you give it